src/main/java/frc/robot/commands/ArmToGoalCommand.java

```diff
@@ -26,7 +28,11 @@ public class ArmToGoalCommand extends CommandBase {
 
     public ArmToGoalCommand(Systems systems, PresetPosition presetPosition, int flags) {
         this(systems, presetPosition.getWristPos(), flags);
-        this.wristDegrees = presetPosition.getWrist();
+
+        //Untested, but i think it'll work
+        if((flags & IGNORE_ROTATION) == IGNORE_ROTATION) {
```

Comment:
a) this is inverted
b) if you wanted it to ignore rotation, just use the constructor below that accepts a Translation2d instead of the PresetPosition - you can use `PresetPosition#getGoal` if you have the PresetPosition instead.